utils/playbackProfiles/helpers/mp4AudioFormats.ts

```diff
@@ -107,7 +161,7 @@ export function getSupportedMP4AudioCodecs(
     codecs.push('dts');
   }
 
-  if (browserDetector.isTv()) {
+  if (browserDetector.isTizen() || browserDetector.isWebOS()) {
```

Comment:
What about TVs with neither, are we planning improvements on that side?

Reply:
There aren't really any actively supported smart TV OSes aside from these two and Android TV. All the others are unsupported.